app/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesService.scala

```diff
@@ -41,30 +42,30 @@ class PlatformInitiativesService @Inject()(
     val initiatives = Seq(
       createDefaultBranchInitiative(
         initiativeName        = "Update Default Branch Terminology",
-        initiativeDescription = s"To update default branch names - [Default Branch Tracker](https://catalogue.tax.service.gov.uk/defaultbranch?name=&teamNames=$teamName&defaultBranch=master) | [Confluence](https://confluence.tools.tax.service.gov.uk/display/TEC/2021/10/08/Default+Branch+Migration%3A+How+To).",
         team                  = team,
+        initiativeDescription = s"To update default branch names - [Default Branch Tracker](" + url"https://catalogue.tax.service.gov.uk/defaultbranch?name=&teamNames=$teamName&defaultBranch=master" + ") | [Confluence](" + url"https://confluence.tools.tax.service.gov.uk/display/TEC/2021/10/08/Default+Branch+Migration%3A+How+To" + ").",
         completedLegend       = "Updated",
         inProgressLegend      = "Master"
       ),
       createUpgradeInitiative(
         initiativeName        = "Play 2.6 upgrade",
-        initiativeDescription = s"Play 2.6 upgrade - Deprecate [Play 2.5 and below](https://catalogue.tax.service.gov.uk/dependencyexplorer/results?group=com.typesafe.play&artefact=play&team=$teamName&flag=production&scope=compile&versionRange=%5B0.0.0%2C2.6.0%5D&asCsv=false) | [Confluence](https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=275944511).",
+        initiativeDescription = s"Play 2.6 upgrade - Deprecate [Play 2.5 and below](" + url"https://catalogue.tax.service.gov.uk/dependencyexplorer/results?group=com.typesafe.play&artefact=play&team=$teamName&flag=production&scope=compile&versionRange=[0.0.0,2.6.0" + "\\)" + "&asCsv=false" + ") | [Confluence](" + url"https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=275944511" + ").",
```

Comment:
do we need the `\\`? just `versionRange=[0.0.0,2.6.0)`

Reply:
The `\\` is for the benefit of the frontend. The frontend uses Laika for the markdown util. Without escaping that character specifically, our `markdown-loader` will interpret that `)` as the closing bracket for the url.

Reply:
can we do the markdown escaping after creating a valid escaped url? e.g. `url"...versionRange=[0.0.0,2.6.0)&...".replace(")","\\)") // escaping required for markdown-loader`